changelogs/fragments/56045-nios-member-module.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+ - Bug fixes to nios_member module
```

Comment:
Note that changelogs are meant for end users so it's better to tell people what the symptoms of the bug that's being fixed had.  (Not a blocker for merge)